src/observers/telemetryObserver.ts

```diff
@@ -1,4 +1,4 @@
-import { BaseEvent, UserSignInTriggered, UserSignInCompleted, UserSignInSucceeded, UserSignInFailed, UserSignOutTriggered, UserSignOutCompleted, BuildTriggered, BuildCompleted, BuildSucceeded, BuildFailed, BuildCacheSizeCalculated, LearnMoreClicked, QuickPickTriggered, QuickPickCommandSelected, DependencyInstallStarted, DependencyInstallCompleted, PackageInstallCompleted } from '../common/loggingEvents';
+import { BaseEvent, UserSignInTriggered, UserSignInCompleted, UserSignInSucceeded, UserSignInFailed, UserSignOutTriggered, UserSignOutCompleted, BuildTriggered, BuildCompleted, BuildSucceeded, BuildFailed, BuildCacheSizeCalculated, LearnMoreClicked, QuickPickTriggered, QuickPickCommandSelected, DependencyInstallStarted, DependencyInstallCompleted, PackageInstallCompleted, CredentialRetrieveFromLocalCredentialManager } from '../common/loggingEvents';
```

Comment:
`CredentialRetrieveFromLocalCredentialManager` -> `CredentialRetrievedFromLocalCredentialManager`

Reply:
will rename it in another PR to keep this PR small.

---
In reply to: [384222478](https://github.com/docascode/vscode-docs-build/pull/70#discussion_r384222478) [](ancestors = 384222478)